yt_dlp/extractor/manyvids.py

```diff
@@ -1,114 +1,98 @@
-import re
 
 from .common import InfoExtractor
+from .. import traverse_obj
 from ..utils import (
     determine_ext,
-    extract_attributes,
     int_or_none,
-    str_to_int,
     url_or_none,
-    urlencode_postdata,
 )
 
 
 class ManyVidsIE(InfoExtractor):
-    _WORKING = False
+    _WORKING = True
     _VALID_URL = r'(?i)https?://(?:www\.)?manyvids\.com/video/(?P<id>\d+)'
-    _TESTS = [{
-        # preview video
-        'url': 'https://www.manyvids.com/Video/133957/everthing-about-me/',
-        'md5': '03f11bb21c52dd12a05be21a5c7dcc97',
-        'info_dict': {
-            'id': '133957',
-            'ext': 'mp4',
-            'title': 'everthing about me (Preview)',
-            'uploader': 'ellyxxix',
-            'view_count': int,
-            'like_count': int,
+    _TESTS = [
+        {
```

Comment:
I saw the reviewer on the prior PR asked for the test to be skipped instead of removed, so I've done that here.